bundles/statistics/statsgrid2016/components/IndicatorParametersList.js

```diff
@@ -114,10 +104,12 @@ Oskari.clazz.define('Oskari.statistics.statsgrid.IndicatorParametersList', funct
         }
         return id;
     },
+    resetMyIndicatorTable: function () {
+        this.getElement().find('.my-indicator').empty();
```

Comment:
This removes the table or resets the cells where user can input data?

Reply:
It removes the indicators list created by the user.

Reply:
The datasets set by setDatasets()? Why not just call setDatasets() without any param or with an empty list?